lang/allocator/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[package]
+name = "sgx-std-lang-allocator"
+version = "0.1.0"
+edition = "2021"
+authors = ["MobileCoin"]
+rust-version = "1.62.1"
+license = "Apache-2.0"
+readme = "README.md"
+repository = "https://github.com/mobilecoinfoundation/sgx-std"
+description = "Allocator for SGX enclave"
```

Comment:
Up for suggestions on how to better differentiate this and the `mc-sgx-alloc` crate.  Currently they have the same description

Reply:
I think "Global Allocator Language Item for SGX Enclaves" is probably the way to go.